pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -95,6 +98,7 @@ type DomainManager interface {
 	GetUsers() ([]v1.VirtualMachineInstanceGuestOSUser, error)
 	GetFilesystems() ([]v1.VirtualMachineInstanceFileSystem, error)
 	SetGuestTime(*v1.VirtualMachineInstance) error
+	GetLoopbackAddress() string
```

Comment:
Why the loopback getter belongs to the DomainManager? Is this just for allowing to inject mocks?

Reply:
yes, i now re-factored it, see please last commit